docs/cellxgene_census_schema.md

```diff
@@ -833,12 +851,42 @@ Cell metadata MUST be encoded as a `SOMADataFrame` with the following columns:
   <tr>
     <td>tissue</td>
   </tr>
+  <tr>
+    <td>nnz</td>
+    <td>int64</td>
+    <td>For this observation, the number of non-zero measurements in the `X['raw']` counts (raw) matrix.</td>
+  </tr>
+  <tr>
+    <td>n_measured_vars</td>
+    <td>int64</td>
+    <td>For this observation, the number of features present in the source H5AD (sum of feature presence matrix).</td>
+  </tr>
+  <tr>
+    <td>raw_sum</td>
+    <td>float32</td>
+    <td>For this observation, the sum of the `X['raw']` counts (raw) matrix values.</td>
+  </tr>
+  <tr>
+    <td>raw_mean_nnz</td>
+    <td>float32</td>
+    <td>For this observation, the mean of the `X['raw']` counts (raw) matrix values.</td>
```

Comment:
```suggestion
    <td>For this observation, the mean of the counts (raw) matrix values. Zeroes are excluded from the calculation.</td>
```

Reply:
to confirm, you want to delete `X['raw']` here ,but not in `raw_variance_nnz`?

Reply:
I updated the comment, probably an artifact of adding the suggestion in a previous version of the file

Reply:
Great - I'll push an update. TY!